content/Solutions/RemoteAccess.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Accessing TrueNAS Remotely"
+description: "Basic information on the variety of ways TrueNAS can be accessed remotely"
+weight: 10
+tags:
+- gettingstarted
+- vpn
+- remote
+- apps
+---
+
+## Overview
+
+TrueNAS is fundamentally a storage appliance, and as such it is recommended to always be run not directly exposed to the internet.
+
+However, there are many instances in which a user will need to access their storage data or services remotely. In most corporate
+environments this is controlled by having access to a corporate VPN service where proper auditing and security best practices
+can be enforced. For the small business or home lab where a corporate VPN isn't available, there are many easier to deploy and
+manage services that can be used to give yourself or your clients access to your storage services and applications. This guide
+will introduce you to several of these and provide a basic walk-through of how to enable remote access to your TrueNAS.
```

Comment:
Suggested rewording to make this active voice/present tense, remove contractions, and to make the text more concise.
```suggestion
However, there are many instances where a user needs to access their storage data or services remotely. 
In most corporate environments, a corporate VPN service controls internal and external access, performs proper auditing, and applies and enforces security best practices. 
For the small business or home lab, where a corporate VPN is not an option, you can choose from many easy-to-deploy and manage services that give you or your clients access to your storage services and applications. 
This guide introduces several access options and provides a basic walk-through enabling remote access to your TrueNAS.
```